x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -169,10 +169,10 @@ describe('Task Runner', () => {
           "action": "execute",
         },
         "kibana": Object {
-          "namespace": undefined,
           "saved_objects": Array [
             Object {
               "id": "1",
+              "namespace": undefined,
```

Comment:
huh.  So this is kinda funny.  I think it's fine for us to have `namespace: undefined` here, conceptually no different than having NO `namespace` property at all, especially when written as an es document ... but still slightly weird.

Reply:
Filtered out now 👍